include/rocksdb/table.h

```diff
@@ -101,14 +101,18 @@ struct BlockBasedTableOptions {
   IndexType index_type = kBinarySearch;
 
   // The index type that will be used for the data block.
-  // The kDataBlockHashSearch index type is not yet implemented.
   enum DataBlockIndexType : char {
     kDataBlockBinarySearch = 0,  // traditional block type
-    kDataBlockHashSearch = 1,     // additional hash index appended to the end.
+    kDataBlockBinaryAndHash = 1,     // additional hash index appended to the end.
```

Comment:
Why the change of name?

Reply:
I gave this suggestion. I feel like having two options "binary" and "hash" may be confusing, as people might think there isn't a binary searchable index and range lookup will also go to hash index.